tests/functional/t/trailing_comma_tuple_9608.rc

```diff
@@ -0,0 +1,5 @@
+[MAIN]
+disable=trailing-comma-tuple
+
+[testoptions]
+exclude_from_minimal_messages_config=True
```

Comment:
Got very confused by that error at first (Didn't see it was a specific automated test that was failing). I chose the fast fix but maybe taking into account the conf file after disabling everything and checking what's expected in the functional test would be better than the current behavior.